### PR TITLE
ai-core-aiapi-postman-train: update explanation

### PR DESCRIPTION
**Pull request by LunaticMaestro**

### CODE REVIEW

File: tutorials/ai-core-aiapi-postman-train/ai-core-aiapi-postman-train.md [hunk 1]
```diff
@@ -17,12 +17,21 @@ author_profile: https://github.com/dhrubpaul
 
 ---
 
-[ACCORDION-BEGIN [Step 1: ](API to list available scenarios)]
+[ACCORDION-BEGIN [Step 1: ](List scenarios)]
 
-Scenarios refers to the use case. It is created by an identifier within the workflows *(YAML files, see screenshots below)* that you synced with GitHub in previous tutorial. For all workflows of same use case, write the same scenario id.
+Scenarios refers to an ML use case. It gets automatically created when you upload workflows *(YAML files, see screenshots below)* to your GitHub repository connected with your SAP AI Core.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-aiapi-postman-train/ai-core-aiapi-postman-train.md:22]: Please fix grammar.

File: tutorials/ai-core-aiapi-postman-train/ai-core-aiapi-postman-train.md [hunk 2]
```diff
@@ -17,12 +17,21 @@ author_profile: https://github.com/dhrubpaul
...
+
+Following are the annotations within your training workflow `training_workflow_tutorial.yaml` that define a scenario.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-aiapi-postman-train/ai-core-aiapi-postman-train.md:24]: Please fix grammar. It is only one workflow.

File: tutorials/ai-core-aiapi-postman-train/ai-core-aiapi-postman-train.md [hunk 3]
```diff
@@ -17,12 +17,21 @@ author_profile: https://github.com/dhrubpaul
...
 
 !![scenario definition inside worflows](img/training/scenario.png)
 
+1. `name` refers to the **Executable ID**, an identifier for your workflow. This must be unique for each executable even if they are part of same the scenario.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-aiapi-postman-train/ai-core-aiapi-postman-train.md:28]: Please fix word order.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by LunaticMaestro** [comment]
@helenaaaaaaaaaa Can you make the grammar edits as suggested.

File: tutorials/ai-core-aiapi-postman-train/ai-core-aiapi-postman-train.md (hunk 2 above)
Comment (jmmargo) [reply tutorials/ai-core-aiapi-postman-train/ai-core-aiapi-postman-train.md:24]: I assume you are referring to "created when you upload workflows" which should then read "created when you upload a workflow" or "created when you upload the workflows"

File: tutorials/ai-core-aiapi-postman-train/ai-core-aiapi-postman-train.md (hunk 3 above)
Comment (jmmargo) [reply tutorials/ai-core-aiapi-postman-train/ai-core-aiapi-postman-train.md:28]: "they are part of the same scenario" instead of "same the scenario"

**Comment by helenaaaaaaaaaa** [comment]
done

File: tutorials/ai-core-aiapi-postman-train/ai-core-aiapi-postman-train.md (hunk 1 above)
Comment (helenaaaaaaaaaa) [reply tutorials/ai-core-aiapi-postman-train/ai-core-aiapi-postman-train.md:22]: done

File: tutorials/ai-core-aiapi-postman-train/ai-core-aiapi-postman-train.md (hunk 2 above)
Comment (helenaaaaaaaaaa) [reply tutorials/ai-core-aiapi-postman-train/ai-core-aiapi-postman-train.md:24]: done

File: tutorials/ai-core-aiapi-postman-train/ai-core-aiapi-postman-train.md (hunk 3 above)
Comment (helenaaaaaaaaaa) [reply tutorials/ai-core-aiapi-postman-train/ai-core-aiapi-postman-train.md:28]: done

**Comment by jmmargo** [comment]
The pull request associated with this tutorial has been closed.  You will need to reopen it for me to commit it to the master.

Thanks,
Joshua

**Pull request closed without merging**